drivers/filesystems/btrfs/btrfs.c

```diff
@@ -288,6 +288,9 @@ static void __stdcall do_xor_basic(uint8_t* buf1, uint8_t* buf2, uint32_t len) {
     uint32_t j;
 
 #if defined(_ARM_) || defined(_ARM64_)
+#ifdef __REACTOS__
```

Comment:
I'd prefer if you would add dummy implementations with a breakpoint for the 3 arm intrinsics instead.